.mock.env

```
@@ -1 +1 @@
-APP_ENV=development
\ No newline at end of file
+APP_API_URL=
```

APP을 prefix로 붙여주신 이유가 궁금합니다

```suggestion
API_URL=
```

---

접근해서는 안되는 환경변수에 클라이언트가 접근하는 것을 최대한 막기 위해서 규칙을 설정해줬습니다.

현재 `env-cmd` 패키지를 사용해서 .env 파일 내 환경 변수들을 주입시켜주고 있습니다만 환경에 따라서 주입 이전에도 여러 환경 변수가 존재할 수 있습니다. 이러한 환경변수에 혹시라도 클라이언트 측에서 접근할 경우 보안적인 이슈가 발생할 수 있기 때문에 최소한의 룰을 설정해주었습니다.